validator/lib/validateBoundingVolume.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
```

Comment:
As mentioned offline, eventually this file should be merged into `validateTileset`.